content/docs/setup/kubernetes/additional-setup/mesh-expansion/index.md

```diff
@@ -260,15 +263,17 @@ particular service, as in the following example.
     EOF
     {{< /text >}}
 
-1. Workloads in a Kubernetes cluster need a DNS mapping to resolve the service running on the VM.
+1. The workloads in a Kubernetes cluster need a DNS mapping to resolve the the domain names of VM services. To
+    integrate the mapping with you own DNS system, use `istioctl register` and creates a Kubernetes `selector-less`
+    service, for example:
```

Comment:
Thanks for accepting my suggestion but please remove lines 269-270.

Reply:
Done.